Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -275,6 +275,12 @@ const ScrollView = createReactClass({
      * @platform ios
      */
     minimumZoomScale: PropTypes.number,
+     /**
+     * Enables nested scrolling for Android API level 21+.
```

Comment:
Could you also open a PR on https://github.com/facebook/react-native-website to add this prop?

Reply:
Added to https://github.com/facebook/react-native-website/pull/249